### PR TITLE
Added functionality for Wallet and Coin classes

### PR DESCRIPTION
**Pull request by iNigoh**
Hey Crypto-bros! ;)

I've added the functionality for Coin and Wallet classes, can you review them! Thx <3

### CODE REVIEW

**Comment by ianrecke** [comment]
Wow, it's really cool!😎

File: app/Domain/Wallet.php [hunk 1]
```diff
@@ -1 +1,90 @@
...
+        $this->wallet_id = $wallet_id;
+        $this->coins = array(); //we create a new empty array where there will be many coins
+        $this->amount = array();
+    }
+
+    /**
+     * @param int $coinPosition
+     * @return float
+     */
+    public function getAmount(int $coinPosition): float
```
Comment (inigo99) [inline app/Domain/Wallet.php:23]: $coinPosition should be $coin_position, to continue with the typing

File: app/Domain/Wallet.php [hunk 2]
```diff
@@ -1 +1,90 @@
...
+    {
+        return $this->amount[$coinPosition];
+    }
+
+    /**
+     * @param int $coinPosition
+     * @param float $amount
+     */
+    public function setAmount(int $coinPosition, float $amount): void
```
Comment (inigo99) [inline app/Domain/Wallet.php:32]: Same as above

File: app/Domain/Wallet.php [hunk 3]
```diff
@@ -1 +1,90 @@
...
+    public function setWalletId(string $wallet_id): void
+    {
+        $this->wallet_id = $wallet_id;
+    }
+
+    /**
+     * @param int $coinPosition
+     * @return Coin
+     */
+    public function getCoins(int $coinPosition): Coin
```
Comment (inigo99) [inline app/Domain/Wallet.php:57]: Same as above

File: app/Domain/Wallet.php [hunk 4]
```diff
@@ -1 +1,90 @@
...
+    {
+        return $this->coins[$coinPosition];
+    }
+
+    /**
+     * @param int $coinPosition
+     * @param Coin $coin
+     */
+    public function setCoins(int $coinPosition, Coin $coin): void
```
Comment (inigo99) [inline app/Domain/Wallet.php:66]: Same as above

**Review by inigo99 — CHANGES_REQUESTED** [review]
Check the names please, everything else amazing AF 🥇

**Review by inigo99 — APPROVED** [review]
Okay, let's go!

**Comment by ianrecke** [comment]
all changes are ok!

**Pull request merged**